docs/hydra/5min-tutorial.md

```diff
@@ -74,32 +76,40 @@ $ docker-compose -f quickstart.yml exec hydra \
 
 OAuth2 client my-client
 OAuth2 client secret: secret
+```
+
+Let's perform the client credentials grant:
 
-# Let's perform the client credentials grant.
+```
 $ docker-compose -f quickstart.yml exec hydra \
     hydra token client \
     --endpoint http://127.0.0.1:4444/ \
     --client-id my-client \
     --client-secret secret
 
 UDYMha9TwsMBejEvKfnDOXkhgkLsnmUNYVQDklT5bD8.ZNpuNRC85erbIYDjPqhMwTinlvQmNTk_UvttcLQxFJY
+```
+
+Let's perform token introspection on that token. Make sure to copy the token you
```

Comment:
Were there reasons why you had this narrative as comments inside the code blocks? I think they make more sense outside of them, as normal Markdown text.

Reply:
No that's definitely a bug :)